release-notes/app-catalog/app-platform-announcement.md

```diff
@@ -0,0 +1,51 @@
+# Announcing Giant Swarm App Platform
+We are pleased to announce that the Giant Swarm App Platform is generally available.
+
+In the past few months, you may have noticed that we have been working on this. We even answered some questions we received as a result of the preview. This has helped us shape the platform and bring it to where it is today.
+
+There are two ways to benefit:
+
+* One, install the apps in the Giant Swarm Catalog on your clusters. Some apps we are working towards fully supporting are NGINX Ingress Controller, Kong, and Prometheus Operator.
```

Comment:
```suggestion
* One, install the apps from the Giant Swarm Catalog on your clusters. Some apps we are working towards fully supporting are NGINX Ingress Controller, Kong, and Prometheus Operator.
```

NGINX is already fully supported, so this section might confuse people that we downgraded our support there. Maybe separate out NGINX

Reply:
+1 it reads like nginx isn't fully supported. this will definitely confuse customers.

What about Aqua?

Reply:
Aqua is in limbo whether Adidas will continue using it. We haven't decided as a team, but the current sentiment seems to be that the only reason to provide it as a managed app is b/c of Adidas. So if Adidas in the end decides to bail, that won't be a a managed app anymore. I don't want to encourage people to ask for Aqua so did not mention here. Same for EFK. Maybe just mention NGINX Ingress Controller and Kong? WDYT?